djangocms_content_expiry/admin.py

```diff
@@ -295,6 +295,17 @@ def get_readonly_fields(self, request, obj=None):
             return self.readonly_fields + ('compliance_number',)
         return self.readonly_fields
 
+    def get_fieldsets(self, request, obj=None):
+        """
+        Form field ordering should remain constant when compliance number is read-only
+        :param request: Request object
+        :param obj: Content expiry object
+        :return: Form fieldsets
+        """
+        fieldsets = super(ContentExpiryAdmin, self).get_fieldsets(request, obj)
+        fieldsets[0][1]['fields'] = ['compliance_number', 'created_by', 'version', 'expires']
```

Comment:
Will this require updating every time fields are updated/added to content expiry? If so, should this not use a constant or something similar?

Reply:
Moved fieldset list to a constant